commands/account/auth.ts

```diff
@@ -0,0 +1,212 @@
+import { Argv, ArgumentsCamelCase } from 'yargs';
+import {
+  loadConfig,
+  getConfigPath,
+  configFileExists,
+  updateAccountConfig,
+  writeConfig,
+  createEmptyConfigFile,
+  deleteEmptyConfigFile,
+  getConfigDefaultAccount,
+} from '@hubspot/local-dev-lib/config';
+import { getAccountIdentifier } from '@hubspot/local-dev-lib/config/getAccountIdentifier';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import {
+  getAccessToken,
+  updateConfigWithAccessToken,
+} from '@hubspot/local-dev-lib/personalAccessKey';
+import { ENVIRONMENTS } from '@hubspot/local-dev-lib/constants/environments';
+import { toKebabCase } from '@hubspot/local-dev-lib/text';
+import { Environment } from '@hubspot/local-dev-lib/types/Config';
+import { CLIAccount } from '@hubspot/local-dev-lib/types/Accounts';
+import { PERSONAL_ACCESS_KEY_AUTH_METHOD } from '@hubspot/local-dev-lib/constants/auth';
+import { DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME } from '@hubspot/local-dev-lib/constants/config';
+
+import { addGlobalOptions, addTestingOptions } from '../../lib/commonOpts';
+import { handleExit } from '../../lib/process';
+import { debugError } from '../../lib/errorHandlers/index';
+import { i18n } from '../../lib/lang';
+import { trackCommandUsage, trackAuthAction } from '../../lib/usageTracking';
+import { personalAccessKeyPrompt } from '../../lib/prompts/personalAccessKeyPrompt';
+import { cliAccountNamePrompt } from '../../lib/prompts/accountNamePrompt';
+import { setAsDefaultAccountPrompt } from '../../lib/prompts/setAsDefaultAccountPrompt';
+import { EXIT_CODES } from '../../lib/enums/exitCodes';
+import { uiFeatureHighlight } from '../../lib/ui';
+import { CommonArgs, ConfigArgs } from '../../types/Yargs';
+
+const i18nKey = 'commands.account.subcommands.auth';
+
+const TRACKING_STATUS = {
+  STARTED: 'started',
+  ERROR: 'error',
+  COMPLETE: 'complete',
+};
+
+async function createOrUpdateConfig(
```

Comment:
Does this function actually create a config? I might be misunderstanding, but from what I can tell, it just updates existing configs and doesn't do anything if `doesConfigExist` is false